include/recognition/planar/IPlanarGraphRecognition.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * IPlanarGraphRecognition.hpp
+ *
+ *  Created on: 24.03.2024
+ *      Author: Dzianis Lahunou
+ *      Ported by: Krzysztof Turowski (krzysztof.szymon.turowski@gmail.com)
```

Comment:
You don't need my name here and elsewhere, you will commit the code to this repo by yourself :)